Resources/Prototypes/XenoArch/artifact_triggers.yml

```diff
@@ -150,6 +193,49 @@
   - type: ArtifactPressureTrigger
     minPressureThreshold: 50
 
+- type: artifactTrigger
+  id: TriggerStimulant
+  targetDepth: 3
+  triggerHint: artifact-trigger-hint-drugs
+  components:
+  - type: Reactive
+    groups:
+      Acidic: [ Touch ]
+    reactions:
+    - reagents: [ Desoxyephedrine, Ephedrine, Stimulants ]
+      methods: [ Touch ]
+      effects:
+      - !type:ActivateArtifact
+
+- type: artifactTrigger
+  id: TriggerPsychedelic
+  targetDepth: 3 # D3s are due to illegality/restrictions of some reactants
+  triggerHint: artifact-trigger-hint-drugs
+  components:
+  - type: Reactive
+    groups:
+      Acidic: [ Touch ]
+    reactions:
+    - reagents: [ Impedrezene, SpaceDrugs, Bananadine, Happiness, HeartbreakerToxin, MindbreakerToxin, Pax ]
```

Comment:
Add THC, remove HeartbreakerToxin.

Reply:
main reason for heartbreaker was it says its a hallucinogen in the guidebook desc